docroot/modules/custom/mass_fields/src/MassUrlReplacementService.php

```diff
@@ -265,17 +289,42 @@ public function processLink($link) {
           // Replace the href with a media URL.
           // Note: We always want to concat download string to the URL.
           $mediaUrl = $mediaEntity->toUrl()->toString() . '/download';
+          if (isset($url['query'])) {
+            $mediaUrl .= '?' . $url['query'];
+          }
           $link = $mediaUrl;
           $changed = TRUE;
         }
+        else {
+          // We cover the case when the file was used in revisions.
+          $mediaQuery = \Drupal::entityQuery('media')
```

Comment:
@arthurbaghdas  why don't use DI here? 

You already have `$this->entityTypeManager` injected, so use it. 

```
$entity_storage = $this->entityTypeManager->getStorage('media');
$entity_ids = $entity_storage->getQuery()
```

Reply:
Good catch, fixed, thank you